### ISSUE TITLE
Objective of this repos

### ISSUE THREAD

**Issue by rickmak** [SkygearIO/skygear-doc-tools#2]
I have a discussion with @howawong on the objective and the way to achieve the objective.

1. The objective is to reduce the need for modifying different repos for the task like: `Adding GA to doc site`
2. `Makefile` already exists across all most all skygear repos.
3. We don't want dev to load yet another tool into his mental model

So I am suggesting instead of making a python project to solve point 1. We can make the project to holding the reusing `Makefile`(aka. common.mk), and it contains a script to populating the change to all related skygear repos.

**Comment by howawong** [comment]
Agree. I will make a generator instead. Sorry for the prolonged PR in py-skygear.
